docs/administration/installation.md

```diff
@@ -214,3 +214,10 @@ Set the `file_type_id` (not possible with Attachment API):
 ```shell
 cv api4 File.update +v file_type_id:name=funding_payment_instruction_template +w 'id = {FILE_ID}'
 ```
+
+## Time Zone
+
+All requests must be done with the same time zone. This means that all CiviCRM
```

Comment:
Does "All requests" refer to all api requests? Or is it referring to something else?

Reply:
Yes, all API requests, independent on how they are initiated (remote API, browser of CiviCRM user, or CLI). (In theory it would be possible execute code in CiviCRM without API calls, though in practice there will be always API calls involved.) Because readers might not associate  browser requests with API calls I've chosen the more general term.

I think I should replace `done` with `executed`.